js/collaboration/collaboration.js

```diff
@@ -24,10 +24,11 @@ class Collaboration {
         this.attempts = 0;
         this.socket = null;
         this.blockList = this.activity.blocks.blockList;
-        this.HOST = process.env.COLLAB_HOST;
         this.PORT = "8080";
+        this.COLLAB_HOST = "http://127.0.0.1";
```

Comment:
MB and collaboration server running as different services means this won't work because they have different IPs, but are within the same subnet and MB has just port 3000 exposed so collaboration server isn't running where MB is running, it's running somewhere else with 8080 exposed.